plugins/jobs/src/js/data/__tests__/JobModel-test.js

```diff
@@ -587,6 +589,42 @@ describe("JobModel Resolver", () => {
       })
     );
 
+    it(
+      "returns the path (job has path)",
```

Comment:
🎨I would like us to write more meaningful statements in general, sth like "a job with a path returns path segments"

Reply:
Your example conflicts with our convention to write descriptions like this `it("does something…`. Do you have a better idea to name the test?

Reply:
I would say "returns the path when the job has one", but it's not blocking for me